contracts/interface/IStakingPoolV2.sol

```diff
@@ -2,21 +2,22 @@
 pragma solidity 0.8.4;
 
 interface IStakingPoolV2 {
-  error NotInRound();
   error StakingNotInitiated();
   error InvalidAmount();
   error ZeroReward();
   error OnlyAdmin();
   error NotEnoughPrincipal(uint256 principal);
   error ZeroPrincipal();
-  error IsOpened();
+  error IsFinished();
+  error IsClosed();
```

Comment:
How about `Finished()` and `Closed()`?  I usually use the prefix, `is`, with boolean values.

Reply:
Here's a link for the naming conventions: https://docs.soliditylang.org/en/v0.8.11/style-guide.html#naming-conventions.